build/common/worker/healthcheck.sh

```diff
@@ -1,28 +1,29 @@
 #!/bin/bash
+set -o errexit -o allexport -o nolog
 
-export COMMON_SITE_CONFIG_JSON='/home/frappe/frappe-bench/sites/common_site_config.json'
+COMMON_SITE_CONFIG_JSON='/home/frappe/frappe-bench/sites/common_site_config.json'
 
 # Set DB Host and port
-export DB_HOST=`cat $COMMON_SITE_CONFIG_JSON | awk '/db_host/ { gsub(/[",]/,"",$2); print $2}' | tr -d '\n'`
-export DB_PORT=`cat $COMMON_SITE_CONFIG_JSON | awk '/db_port/ { gsub(/[",]/,"",$2); print $2}' | tr -d '\n'`
+DB_HOST=`cat $COMMON_SITE_CONFIG_JSON | awk '/db_host/ { gsub(/[",]/,"",$2); print $2}' | tr -d '\n'`
```

Comment:
since these `awk` commands are repeatedly being used, let's turn it into a function in the file for simplicity.

Reply:
done